src/models/experts/reducers.ts

```diff
@@ -66,6 +70,12 @@ export const expertsSlice = createSlice({
   extraReducers: {
     ...getAsyncActionsReducer(fetchExperts as any),
     ...getAsyncActionsReducer(fetchExpertsAutorsList as any),
+    ...getAsyncActionsReducer(deleteAuthor as any),
+    [deleteAuthor.fulfilled.type]: (state, action) => {
+      state.data.expertIds = state.data.expertIds.filter(
```

Comment:
I'm not sure but I feel like this way you override `[asyncAction.fulfilled.type]` from `getAsyncActionsReducer` so 

```
    state.loading = LoadingStatusEnum.succeeded;
    state[key] = action.payload;
```

is never executed. May you verify that? Sure, when deleting author it does not matter but if my assumption is correct - it may shoot us in different place(and we better to extend https://github.com/ita-social-projects/dokazovi-fe/blob/17ed7fc04b13798a0b4f26e16d3000b8c16db237/src/models/helpers/asyncActions.ts to pass custom logic into `getAsyncActionsReducer` as an argument)

Reply:
You're absolutely right. I redid it in accordance with the logic of work `getAsyncActionsReducer`